apps/web/app/dashboard/app/[name]/appLinks.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
```

Comment:
Use consistent file naming patterns across the project. Change it to `app-links.tsx`.

Reply:
Change the file name to `tabs.tsx`.

Reply:
Changed the file name